tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -142,7 +144,100 @@ def test_brand_string(test_microvm_with_api, network_config):
 @pytest.mark.skipif(
     PLATFORM != "x86_64", reason="CPU features are masked only on x86_64."
 )
-@pytest.mark.parametrize("cpu_template", ["T2", "C3"])
+@pytest.mark.skipif(
+    cpuid_utils.get_cpu_vendor() != cpuid_utils.CpuVendor.INTEL,
+    reason="CPU templates are only available on Intel.",
+)
+@pytest.mark.skipif(
+    utils.get_kernel_version(level=1) not in ["5.10", "4.14"],
```

Comment:
We should use the SUPPORTED_KERNELS file so that we have a centralized place that dictates what are the currently supported kernel versions